HW_1/task_3.py

```diff
@@ -0,0 +1,42 @@
+"""
+Напишите программу, принимающую на вход координаты точки (X и Y) и выдающую номер четверти плоскости,
+в которой находится эта точка (или на какой оси она находится).
+
+Ввод: два значения типа <int>
+Вывод: значение типа <int> либо значение типа <str>
+
+Пример:
+
+34
+-30
+4
+
+2
+4
+1
+
+-34
+0
+Точка на отрицательной части оси абсцисс
+"""
+x = int(input('Введите координату X: '))
+y = int(input('Введите координату Y: '))
+
+if x > 0 and y > 0:
```

Comment:
Это и последующее условия можно свернуть относительно нуля.
```suggestion
if x > 0 < y:
```